pkgs/development/libraries/poppler/default.nix

```diff
@@ -34,13 +34,13 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "poppler-${suffix}";
-  version = "21.06.1"; # beware: updates often break cups-filters build, check texlive and scribusUnstable too!
+  version = "21.12.0"; # beware: updates often break cups-filters build, check texlive and scribusUnstable too!
```

Comment:
Where does packages checked?

Reply:
What do you mean?

Reply:
Did we check if cups-filters, texlive and scribus still build?